sdk/textanalytics/ai-text-analytics/src/textAnalysisClient.ts

```diff
@@ -493,4 +512,245 @@ export class TextAnalysisClient {
       }
     );
   }
+
+  /**
+   * Performs an array (batch) of actions on the input documents. Each action has
+   * a `kind` field that specifies the nature of the action. See ${@link AnalyzeBatchActionNames}
+   * for a list of supported actions. In addition to `kind`, actions could also
+   * have other parameters such as `disableServiceLogs` and `modelVersion`.
+   *
+   * The results array contains the results for those input actions where each
+   * item also has a `kind` field that specifies the type of the results.
+   *
+   * See {@link https://docs.microsoft.com//azure/cognitive-services/language-service/concepts/data-limits}
+   * for data limits.
+   *
+   * ### Examples
+   *
+   * #### Keyphrase extraction and Pii entity recognition
+   *
+   * ```js
+   * const poller = await client.beginAnalyzeBatch(
+   *  [{ kind: "KeyPhraseExtraction" }, { kind: "PiiEntityRecognition" }],
+   *  documents
+   * );
+   * const actionResults = await poller.pollUntilDone();
+   *
+   * for await (const actionResult of actionResults) {
+   *  if (actionResult.error) {
+   *    throw new Error(`Unexpected error`);
+   *  }
+   *  switch (actionResult.kind) {
+   *    case "KeyPhraseExtraction": {
+   *      for (const doc of actionResult.results) {
+   *        // do something
+   *      }
+   *      break;
+   *    }
+   *    case "PiiEntityRecognition": {
+   *      for (const doc of actionResult.results) {
+   *        // do something
+   *      }
+   *      break;
+   *    }
+   *  }
+   * }
+   * ```
+   *
+   * @param actions - an array of actions that will be run on the input documents
+   * @param documents - the input documents to be analyzed
+   * @param language - the language that all the input strings are
```

Comment:
I wonder if we should mention the format that this string can take. Is it a full language name in english ("spanish")? a locale style string "es-es" or just a 2 letter code "es"? May be worth mentioning if it follows any ISO standard

Reply:
Would it make sense to call this parameter languageCode?

Reply:
I clarified the docstring language and added a link to a list of possible values.